lib/core/CDataFrame.cc

```diff
@@ -208,7 +209,13 @@ void CDataFrame::writeRow(const TWriteFunc& writeRow) {
 }
 
 void CDataFrame::writeCategoricalColumns(TBoolVec columnIsCategorical) {
```

Comment:
I think I was intending to mirror the `writeRow` function, but `categoricalColumns` would be more consistent with most of the codebase and as it stands I can see it could read like write their values. Ok I'll rename.

Reply:
Done in [d83a6b7](https://github.com/elastic/ml-cpp/pull/548/commits/d83a6b792c0db67654f71e58151db0ac916ce3f4).